internal/commands/pull/command_test.go

```diff
@@ -197,15 +200,30 @@ Successfully pulled app down: app
 		})
 
 		t.Run("should return the error when exporting dependencies fails", func(t *testing.T) {
-			profile, teardown := mock.NewProfileFromTmpDir(t, "pull_handler_test")
-			defer teardown()
+			t.Run("when include package json file", func(t *testing.T) {
+				profile, teardown := mock.NewProfileFromTmpDir(t, "pull_handler_test")
+				defer teardown()
 
-			_, ui := mock.NewUI()
+				_, ui := mock.NewUI()
 
-			cmd := &Command{inputs{Project: "elsewhere", LocalPath: "app", IncludeDependencies: true}}
+				cmd := &Command{inputs{Project: "elsewhere", LocalPath: "app", IncludePackageJSON: true}}
 
-			err := cmd.Handler(profile, ui, cli.Clients{Realm: realmClient})
-			assert.Equal(t, errors.New("something bad happened"), err)
+				err := cmd.Handler(profile, ui, cli.Clients{Realm: realmClient})
+				assert.Equal(t, errors.New("something bad happened with package json input"), err)
+
+			})
+			t.Run("when include node modules zip", func(t *testing.T) {
```

Comment:
[nit] add a newline here

and we should dupe/test-table this to add coverage for `IncludeDependencies`